common/unsafe/src/main/java/org/apache/spark/sql/catalyst/util/CollationFactory.java

```diff
@@ -324,6 +325,11 @@ protected enum SpaceTrimming {
 
       private static final int INDETERMINATE_COLLATION_ID = -1;
 
+      /**
+       * UTF8 Binary collation version.
+       */
+      private static final String UTF8_BINARY_VERSION = "1.0";
```

Comment:
If we really depends on ICU here, and by the source code it seems it is true, we should use ICU version for UTF8_BINARY too. @stefankandic WDYT?

Reply:
Agreed